analysis/common_variables.py

```diff
@@ -265,6 +265,27 @@ def generate_common_variables(index_date_variable, exposure_end_date_variable, o
         "tmp_out_date_vascular_dementia_snomed", "tmp_out_date_vascular_dementia_hes", "tmp_out_date_vascular_dementia_death",
     ),
 
+    # Lewy body
+    # Primary
+    tmp_out_date_lewy_body_dementia_snomed=patients.with_these_clinical_events(
```

Comment:
Temporary variables are only necessary if you have code lists from more than one coding system. The approach you have taken requires two variables to be extracted and then loaded when one variable is sufficient. Please call this variable `out_date_lew_body_dementia` and remove the `patients.minimum_of()` call below.

Reply:
Please implement the same fix throughout the code - for instance, cognitive impairment and restless leg syndrome are also coded this way.

Reply:
Changes have been implemented.